plugins/search-backend/api-report.md

```diff
@@ -18,5 +19,7 @@ export function createRouter(options: RouterOptions): Promise<express.Router>;
 export type RouterOptions = {
   engine: SearchEngine;
   logger: Logger_2;
+  discovery: PluginEndpointDiscovery;
+  allowedLocationProtocols?: string[];
```

Comment:
Tbh I don't see much value in making this configurable for now, perhaps just leave as a hardcoded http + https for now and see if a need for something different pops up in the future?

Reply:
Yeah it's not really needed 👍